src/seedu/addressbook/data/person/Person.java

```diff
@@ -83,5 +83,13 @@ public int hashCode() {
     public String toString() {
         return getAsTextShowAll();
     }
+    
+    String getPrintableString(Printable... printables){
```

Comment:
yep this is correct. you can also make Person implement Printable and create an overloading method to call this method. Also, do remember to set the access level of this method.